pegasus/sites.v3/hourofcode.com/views/whats_next.haml

```diff
@@ -3,4 +3,4 @@
   %ul{:style=>"line-height:28px"}
     %li=hoc_s(:whats_next_how_to).gsub('/how-to', resolve_url('/how-to'))
     %li=hoc_s(:whats_next_how_to_promote_link).gsub('/promote', resolve_url('/promote'))
-    %li=hoc_s(:whats_next_try).gsub('https://code.org/learn', resolve_url('https://code.org/learn'))
+    %li=hoc_s(:whats_next_try).gsub('%{learn_url}', resolve_url('/learn'))
```

Comment:
That's a fair point. Let's noticeable bc we don't have the benefits of React forcing us to use dangerouslySetInnerHtml, but I think you're right that these are equally problematic.